apps/prairielearn/src/lib/editors.ts

```diff
@@ -113,10 +120,7 @@ export class Editor {
     return serverJob;
   }
 
-  /**
-   * @param {import('./server-jobs.js').ServerJobExecutor} serverJob
-   */
-  async executeWithServerJob(serverJob) {
+  async executeWithServerJob(serverJob: ServerJobExecutor) {
```

Comment:
Not that I really mind, but above we have `job: ServerJob` and here we have `serverJob: ServerJobExecutor`, which feels a little confusing.

Reply:
Fair enough! I'm going to leave this as is for now since CI is green. We can see about tweaking this in a followup PR.